concepticondata/concepticon.tsv

```diff
@@ -3753,3 +3753,17 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3767	RICE SEEDLING	Agriculture and vegetation	The seedling of a rice plant.	Person/Thing	
 3768	POUND (FOOD)	Motion	To prepare food with a pounding or beating movement, for example, pre-processing of grain or of sago.	Action/Process	
 3769	GRAPEVINE	Agriculture and vegetation	A climbing plant that produces grapes and belongs to the family Vitaceae.	Person/Thing	
+3770	YOU TWO (OBLIQUE CASE)	Kinship	Oblique cases of the second person dual pronoun.	Other	
+3771	THEM TWO	Kinship	Oblique cases of the third person dual pronoun.	Other	
+3772	THEIR (DUAL)	Kinship	Belonging to the third person dual.	Other	
+3773	YOU (MORE THAN TWO)	Kinship	Second person plural excluding dual pronoun.	Other	
+3774	THEM (MORE THAN TWO)	Kinship	Oblique cases of the third person plural excluding dual pronoun.	Other	
+3775	US TWO	Kinship	Oblique cases of the first person dual pronoun.	Other	
+3776	HIM OR HER OR IT	Kinship	Oblique cases of the third person singular pronoun.	Other	
+3777	YOU (MORE THAN TWO OBLIQUE CASE)	Kinship	Oblique cases of the second person plural excluding dual pronoun.	Other	
+3778	OUR (DUAL)	Kinship	Belonging to the first person dual.	Other	
+3779	OUR (MORE THAN TWO)	Kinship	Belonging to the first person plural excluding dual.	Other	
+3780	THEIR (MORE THAN TWO)	Kinship	Belonging to the third person plural excluding dual.	Other	
+3781	YOUR (MORE THAN TWO)	Kinship	Belonging to the second person plural excluding dual.	Other	
+3782	WE (MORE THAN TWO)	Kinship	First person plural excluding dual pronoun.	Other	
```

Comment:
I'm not sure whether those will be relevant, but what about the combination of dual/plural, case and inclusiveness all in one concept, as Tiago mentioned in #396 ? 
E.g. WE (MORE THAN TWO, INCLUSIVE, OBLIQUE)